.babelrc

```diff
@@ -1,9 +1,9 @@
 {
   "presets": [
-    "es2015",
-    "react"
+    "@babel/env",
```

Comment:
do you mean "@babel/preset-env" here?

I recommend explicitly writing targets - https://babeljs.io/docs/en/babel-preset-env#targets

Otherwise you transform all ECMAScript 2015+ code by default.

Reply:
Great! tnx for picking this up @tabrindle ... not even sure how "@babel/env" this worked.. I will replace to "@babel/preset-env"

Also agree with targets.. I am going to use "node" as its during build time. Do you agree? or let know which else would be better

Reply:
Actually look at this: https://babeljs.io/docs/en/usage

```
npm install --save-dev @babel/core @babel/cli @babel/preset-env

const presets = [
  [
    "@babel/env",
    {
      targets: {
        edge: "17",
        firefox: "60",
        chrome: "67",
        safari: "11.1",
      },
      useBuiltIns: "usage",
    },
  ],
];
```

Maybe "@babel/env", is an alias to "@babel/preset-env"?

Also I have added targets of '"targets": "> 0.25%, not dead"', what do you think?
https://github.com/newbreedofgeek/react-stepzilla/pull/137/commits/7f2804897a1982cc0c4ef5710faba1140c21e114#diff-e56633f72ecc521128b3db6586074d2cR5

Reply:
Experiment with this - these targets are just browserlist queries. https://browserl.ist/?q=%3E0.25%25%2C+not+dead - In this query, not dead makes no change ¯\_(ツ)_/¯. 

Personally I would exclude opera mini and IE 11 (both just hugely problematic), while trying to keep as much mobile compatibility as possible. For instance, iOS 8 and 9 still have surprising amounts of market share. For a library as universal as yours, 0.25 might be a little high of a threshold. 

It doesn't really cost much to move your targets back, but I do recommend setting explicit targets, otherwise people may complain about version support.

Does that help?

Reply:
It does thanks @tabrindle